ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -126,6 +129,14 @@ type RayClusterReconciler struct {
 	Recorder          record.EventRecorder
 	BatchSchedulerMgr *batchscheduler.SchedulerManager
 	IsOpenShift       bool
+
+	headSidecarContainer   *corev1.Container
+	workerSidecarContainer *corev1.Container
+}
+
+type RayClusterReconcilerOptions struct {
+	HeadSidecarContainer   *corev1.Container
+	WorkerSidecarContainer *corev1.Container
```

Comment:
Can these be lists in order to support multiple sidecars?

Reply:
I think that's a good suggestion, I can update. Do you have a use-case you can share though for needing mulitple containers?

Reply:
Not right now. However, asfaik it is normal to have multiple sidecars with each one having a specific responsibility, i.e. one for logs and one for networking